src/oop/object_independence/hard/q3/QuestionText.md

```diff
@@ -0,0 +1,46 @@
+# oop-object_independence-hard-q3
+
+- Java question
+
+## Question Text
+
+Suppose you wanted to model the different kinds of plants present in an environment. Using the requirements below, 
+create two classes, one called `Tree` and its test class `TestFoliage`.
+
+## Tree Class
+
+Your `Tree` class must have the following items:
+
+- attributes:
+    - A `boolean` attribute called **isRare**
+    - A `double` attribute called **height**
+    - Note that none of these attributes should be accessible from another class
+
+- Constructor:
+    - A constructor visible from another class which accepts the parameters `boolean, double` and sets the **isRare** 
+      and **height** attributes with the passed values.
+    - This constructor should be visible to other classes
+
+- Methods (all the following methods should be visible from another class):
+    - A method called `getIsRare` that returns the value of the **isRare** attribute
+    - A method called `getHeight` that returns the value of the **height** attribute
+    - A method called `setIsRare` that accepts an `boolean` as a parameter, updates the **isRare** attribute to the 
+      passed value, and returns nothing
+    - A method called `setHeight` that accepts a `double` as a parameter, updates the **height** attribute to the passed
+      value, and returns nothing
+    - A method called `deforestation` that updates the **height** value to 0
```

Comment:
Please mention that there are no parameters and the return type for this method.